assemble/bin/accumulo-cluster

```diff
@@ -485,6 +485,26 @@ function control_services() {
   if [[ $ARG_ALL == 1 && $ARG_LOCAL == 0 && $operation == "stop" ]]; then
     echo "Stopping Accumulo cluster..."
     if ! isDebug; then
+      # Stop all of the the Scan Server processes
```

Comment:
Later in the file on lines 582 and 612 there is some existing code for stopping scan servers and GC, however it looks a bit different.  Could that code be removed?

Reply:
> The code below is also used for starting or stopping just the gc or sserver processes, and for starting everything.

That clue was very helpful in understanding the bigger picture.  Did not realize this function did start and stop.

Compared this new code to the existing code and it looks good.  Noticed the local var may be declared twice in the same function, not sure if bash cares about that though.

Reply:
bash didn't error for me, but the `gc` and `sserver` var definitions could be moved up to line 484 and the later ones removed to avoid the issue.

Reply:
Yeah, I'm wasn't sure if `local` is relative to a block of code or not in bash.

Reply:
I removed the duplicate local declarations in 4e6bd8b

Reply:
> Yeah, I'm wasn't sure if `local` is relative to a block of code or not in bash.

Looks like it's relative to the "block of code", but that's defined as the contents of an anonymous function.
https://tldp.org/LDP/abs/html/special-chars.html#CODEBLOCKREF